src/main/java/service/filework/AddDataToFile.java

```diff
@@ -0,0 +1,7 @@
+package service.filework;
+
+import java.util.Map;
+
+public interface AddDataToFile {
+    void addInStorage(Map<String, Integer> fruitAndQuantity, String filePath);
```

Comment:
```suggestion
    void write(String filePath);
```
you can get Storage inside of this method directly